src/vs/workbench/api/common/extHostNotebookDocument.ts

```diff
@@ -110,6 +110,14 @@ export class ExtHostCell {
 				output.items.length = 0;
 			}
 			output.items.push(...newItems);
+			if (output.items.every(item => notebookCommon.isTextStreamMime(item.mime))) {
```

Comment:
TODO: This needs to be via a proposed API, have a specific method such as `appendStreamOutputItem` or the like